andela/mazus-ah-backend
# #167165070 Adds Pagination Support For Articles

### What does this PR do?
Adds pagination for articles

#### Description of Task to be completed?
Adds pagination to the get all article route
Adds pagination to get all articles by tags routes
Adds pagination to get all articles by authors

#### How should this be manually tested?
> 1. Fetch this branch on the terminal using 
> `git fetch origin ft-article-pagination-167165070`
> 2. Install dependencies using `npm install`
> 3. Start the server with `npm run server`
> 4. On Postman, sign up by making a GET request to /api/v1/articles. 
> 5 The result should be limited to 10 articles, you can do the same with tags as a query parameter and authors routes /api/v1/articles/:id

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#167165070](https://www.pivotaltracker.com/story/show/167165070)

#### Screenshots (if appropriate)
N/A
